src/script/components/InputBar/components/TypingIndicator/TypingIndicator.tsx

```diff
@@ -59,7 +59,7 @@ const TypingIndicator: FC<TypingIndicatorProps> = ({conversationId}) => {
           />
         ))}
       </div>
-      <span css={indicatorTitleStyles} data-uie-name="typing-indicator-title">
+      <p css={indicatorTitleStyles} data-uie-name="typing-indicator-title">
```

Comment:
why is a p the correct element here?

Reply:
the screen reader on macos is ignoring announcement of the content of span but with P it does correctly read the inside text.